### DISCUSSION TITLE
参政议政从大学生抓起

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#10] [Announcements]
# 参政议政从大学生抓起

我们大学的政治课应当是一门实践课，而不是纸上谈兵。既然政治课不教，我们自学！

https://zexwoo.blog/posts/editorial/democracy/

**Comment by qxsny** [comment]
# I don't think so,.you know,we're following the Chinese Communist Party.
--------------------
## College students are thinking more about their future！Instead of looking for utopias in a system that already has a clear general direction！

**Comment by ghost** [comment]
啊我们政治老师在课上发表tai独言论。。。。。

**Reply by qxsny** [reply]
You must report it to the government!!!
